tiledb/sm/storage_manager/storage_manager.cc

```diff
@@ -268,6 +268,63 @@ StorageManagerCanonical::load_array_schemas_and_fragment_metadata(
       Status::Ok(), array_schema_latest, array_schemas_all, fragment_metadata};
 }
 
+std::tuple<
```

Comment:
Should this API take in an opened array so we don't have to implement this in the StorageManager?

Reply:
I don't think so, it's pretty directly related to its sibling load_array_schemas_and_fragment_metadata(),
and I believe there would be undesirable confusion as to what fragments were or not loaded for that opened array, and the resulting state after all fragments may have been loaded (not possible for normally opened array AFAICT) and that its safer to position this as currently implemented near its sibling.

Reply:
Appears the entire FragmentInfo class/api (including retrieving mbr data) retrieves information without having an opened array (but maybe I missed something.)

Reply:
Same answer as above. We should later fix fragment info to use an opened array but in the meantime should introduce new code with bad behavior.

Reply:
method moved from class StorageManager to class Array, parameters changed accordingly ('this' becomes class Array).